.github/shared/src/spec-model.js

```diff
@@ -65,7 +73,29 @@ export class SpecModel {
             continue;
           }
 
-          const refs = await inputFile.getRefs();
+          let refs;
+          try {
+            refs = await inputFile.getRefs();
+          } catch (error) {
+            if (isResolverError(error)) {
+              const resolverError = /** @type {SpecError} */ {
+                message: `Spec Model encountered error resolving refs for input-file: ${inputFile.path}
```

Comment:
This is interpreting an error from swagger.js. Probably swagger.js should catch the error, wrap it up in something spec-model specific and send it up the stack for further context/interpretation.